cypress_test/integration_tests/common/helper.js

```diff
@@ -921,14 +921,14 @@ function waitUntilIdle(selector, content, waitingTime = mobileWizardIdleTime) {
 // selector - a CSS selector to query a DOM element to wait on to be idle.
 // content - a string, a content selector used by cy.contains() to select the correct DOM element.
 // waitingTime - how much time to wait before we say the item is idle.
-function clickOnIdle(selector, content, waitingTime = mobileWizardIdleTime) {
+function clickOnIdle(selector, content, waitingTime = mobileWizardIdleTime, forceClick = false) {
 	cy.log('Clicking on item when idle - start.');
 
 	waitUntilIdle(selector, content, waitingTime);
 
 	if (content) {
 		cy.contains(selector, content)
-			.click();
+			.click({force: forceClick});
```

Comment:
hmm, this one looks like it will never fail - but in some cases should, for example object is covered by other element.
It should not be changed. Why it was needed?

Reply:
if button was not visible and it failed (in notebookbar) we just do a scroll right in other tests - this can help